src/extensions/default/bramble/lib/ConsoleManager.js

```diff
@@ -0,0 +1,29 @@
+define(function (require, exports, module) {
+    "use strict";
+
+    var BrambleEvents = brackets.getModule("bramble/BrambleEvents");
+    var EditorManager = brackets.getModule("editor/EditorManager");
+    var CommandManager = brackets.getModule("command/CommandManager");
+    var Commands = brackets.getModule("command/Commands");
+
+    var ConsoleManagerRemote = require("text!lib/ConsoleManagerRemote.js");
+    var IFrameBrowser = require("lib/iframe-browser");
+
+    var isConsoleEnabled = false;
+
+    function getRemoteScript() {
+        return "<script>\n" + ConsoleManagerRemote + "</script>\n";
```

Comment:
So @Pomax is giving good advice, but in this case we have to do this a hacky way.  We are preparing the injected code that we put into the preview iframe, and we need the raw source of your remote console script so we can concatenate it to the front of the HTML we're going to preview.

You need to require the text value vs. the module as such, and we do it like this:

https://github.com/mozilla/brackets/blob/master/src/extensions/default/bramble/lib/LinkManager.js#L10

with requirejs' text plugin.